src/inputs/select-card/story-helper.ts

```diff
@@ -219,6 +237,23 @@ export const getSelectCardArgTypes = (): ArgTypes => ({
             type: 'text',
         },
     },
+    bottom: {
```

Comment:
"bottomSlot" would be much safer to prevent duplication with props or event names!